src/Config.ts

```diff
@@ -1,140 +1,174 @@
 import { Node, Link } from './Topology'
 import Tools from './Tools'
 
+const LEVEL_K8S_FEDERATION = 100
+const LEVEL_K8S_CLUSTER = 101
+const LEVEL_K8S_NODE = 102
+const LEVEL_K8S_POD = 103
```

Comment:
Split into two views:

1. k8s deployment view
2. k8s logical view

in (1) we need at least another layer for deployments and not show things that are logical
in (2) we need at least another layer for services and ingress to show end-points

It is ok not to show all objects in all views. so for example I would advice to move the storage out to maybe another view that is based on storage only

Reply:
there were many view in the previous ui:

- storage
- network
- compute
- deployment
- service

this will be addressed in a separate PR